package/aws/snstopic/v1alpha1/composition.yaml

```diff
@@ -0,0 +1,66 @@
+apiVersion: apiextensions.crossplane.io/v1
+kind: Composition
+metadata:
+  name: xawssnstopicv1alpha1s.messaging.xplane.dfds.cloud
+spec:
+  compositeTypeRef:
+    apiVersion: messaging.xplane.dfds.cloud/v1alpha1
+    kind: XAWSSNSTopicV1alpha1
+  patchSets:
+  - name: configname
+    patches:
+    - fromFieldPath: spec.claimRef.namespace
+      toFieldPath: spec.providerConfigRef.name
+      transforms:
+      - type: string
+        string:
+          fmt: "%s-aws"
+      policy:
+        fromFieldPath: Required
+  resources:
+  - name: snstopic
+    base:
+      apiVersion: notification.aws.crossplane.io/v1beta1
+      kind: SNSTopic
```

Comment:
Shouldn't the Kind be Topic from the non-deprecated namespace?

Reply:
you're right! I should have seen that :/ going to update the code